expat/lib/xmlparse.c

```diff
@@ -133,7 +133,7 @@
       * BSD / macOS (including <10.7) (arc4random): HAVE_ARC4RANDOM, \
       * libbsd (arc4random_buf): HAVE_ARC4RANDOM_BUF + HAVE_LIBBSD, \
       * libbsd (arc4random): HAVE_ARC4RANDOM + HAVE_LIBBSD, \
-      * Linux (including <3.17) / BSD / macOS (including <10.7) (/dev/urandom): XML_DEV_URANDOM, \
+      * Linux (including <3.17) / BSD / macOS (including <10.7) (/dev/urandom) / Solaris >=8 (/dev/urandom): XML_DEV_URANDOM, \
```

Comment:
Thanks for the adjustments, but something went wrong here: We have `(/dev/urandom)` here twice now. Please drop the first occurrence. Thanks!